### PR TITLE
Update documentation for relative_time()

### PR DESCRIPTION
**Pull request by rlippmann**
Update documentation for relative_time() to add new parameters is_future and depth allowing for it to use future datetimes and more precision.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/85242
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration/templating.markdown [hunk 1]
```diff
@@ -459,7 +459,7 @@ For example, if you wanted to select a field from `trigger` in an automation bas
...
-- `relative_time` converts datetime object to its human-friendly "age" string. The age can be in second, minute, hour, day, month or year (but only the biggest unit is considered, e.g.,  if it's 2 days and 3 hours, "2 days" will be returned). Note that it only works for dates _in the past_.
+- `relative_time(datetime,is_future=False,depth=1)` converts datetime object to its human-friendly time string. The time string can be in seconds, minutes, hours, days, months and years.  is_future determines if the datetime is in the future or not.  If is_future is left at its default of False, only datetimes in the past may be used, otherwise the object is returned.  If is_future is True, future datetimes may be used.  In this case, if the datetime is in the past, the string "0 seconds" is returned.  depth indicates the number of units returned with the last unit rounded (i.e depth = 1 could return "2 years" while depth = 2 could return "1 year 11 months").  This function can also be used as a filter.
```
Comment (frenck) [inline source/_docs/configuration/templating.markdown:462]: The `relative_time` template function does not take these parameters. The method used only takes a single value.

Could you clarify what this change is about?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/configuration/templating.markdown (hunk 1 above)
Comment (rlippmann) [reply source/_docs/configuration/templating.markdown:462]: Oops, I meant to make the changes against next.

Sorry, my first doc change.

**Comment by rlippmann** [comment]
Sorry, need to rebase this to new.  But I don't know how to do it...

**Comment by rlippmann** [comment]
Created new pull request against next.

**Pull request closed without merging**